forge/files/src/index.js

```diff
@@ -43,13 +26,22 @@ app.on('window-all-closed', () => {
   }
 });
 
-app.on('ready', () => {
+app.on('ready', async () => {
   if (isDev) {
-    require('devtron').install();
-    installExtension(EMBER_INSPECTOR)
-      .catch((err) => console.log('An error occurred: ', err));
+    try {
+      require('devtron').install();
+    } catch (err) {
+      console.log('Failed to install Devtrom: ', err);
```

Comment:
Typo? Devtro**m** -> Devtro**n**

P.S. Thank you for making this message more meaningful. I hadn't personally ran into it, but overly generic error messages drive me nuts in logs.

Reply:
Haha, yeah, I noticed that myself and have it fixed locally (new commit coming soon)